internal/nginx-meshctl/commands/top.go

```diff
@@ -134,34 +133,6 @@ func Top() *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "default", "namespace where the resource(s) resides")
-	cmd.SetUsageTemplate(topTemplate)
 
 	return cmd
 }
-
-// Custom template for TOP to fix the namespace usage statement (default template shows parent usage).
```

Comment:
The reason `top` and `upgrade` had custom templates was because they have flags that override parent flags, and the default usage statements would show the parent usage for those flags, instead of the child usage of those flags.

Reply:
From what I saw, the overriden flags were being printed out in the default usage statement, and not the parent usage. Adding the flag explicitly in the template was leading to the overriden flags being printed twice.